userspace/falco/falco_outputs.cpp

```diff
@@ -268,8 +276,21 @@ inline void falco_outputs::push(const ctrl_msg& cmsg)
 {
 	if (!m_queue.try_push(cmsg))
 	{
-		fprintf(stderr, "Fatal error: Output queue reached maximum capacity. Exiting.\n");
-		exit(EXIT_FAILURE);
+		switch (m_recovery)
```

Comment:
Not sure if we are 100% thread safe here :thinking:

Reply:
Probably not, maybe @jasondellaluce once back may have additional ideas that we could also address in a follow up PR since Jason worked a lot on the thread safety aspects all around.

Reply:
what would happen in multi-source scenarios?